docs/tutorials/estimator_vqe.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>This indeed is taking a long time. I would suggest to reduce to only one bond distance, e.g. at equilibrium distance (I think it's 0.735 A). See https://qiskit.org/documentation/nature/tutorials/03_ground_state_solvers.html</p><p> </p><p>And instead of showing the dissociation curve, we can show the energy vs iterations history.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/235/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Ok, I have updated it to be a single distance.